examples/README.md

```diff
@@ -19,15 +19,19 @@ alias helm-west="KUBECONFIG=$(pwd)/west.kubeconfig helm"
 
 ### Trust model
 
-Currently, mesh federation does not work for meshes using different root certificates, but this is on the roadmap.
+Currently, workloads in federated meshes cannot establish mTLS connections if meshes use different root certificates.
+In such a case, use [SPIRE](https://spiffe.io/docs/latest/spire-about/) and trust domain [federation](https://spiffe.io/docs/latest/architecture/federation/readme/).
+You can follow this [guide](spire/README.md) to see how these solutions work together.
```

Comment:
> In this example, workloads in federated meshes are able to establish mTLS connections because a common root certificate is used.

I do not think this is better. First of all, Istio does not officially support establishing mTLS between workloads using certs signed by different roots, so saying "in this example" suggests that this is valid for this particular setup, but this the general constraint in Istio.